circle.yml

```diff
@@ -2,5 +2,9 @@ dependencies:
   pre:
     - cp .env.example .env
     - bash ./build-elasticsearch.sh
+    - sudo apt-get update; sudo apt-get install libicu52
```

Comment:
I don't think upgrading phantomjs is helping, frankly. I still see DeadClient errors occasionally.

another approach e.g. https://github.com/18F/C2/pull/1043